FF1Lib/FF1Rom.cs

```diff
@@ -81,7 +81,17 @@ public static async Task<FF1Rom> CreateAsync(Stream readStream)
 
 		public void Randomize(Blob seed, Flags flags, Preferences preferences)
 		{
-			var rng = new MT19337(BitConverter.ToUInt32(seed, 0));
+			MT19337 rng;
+			using (SHA256 hasher = SHA256.Create())
+			{
+				string flagstring = Flags.EncodeFlagsText(flags);
+				byte[] FlagsBytes = hasher.ComputeHash(Encoding.UTF8.GetBytes(flagstring));
+				byte[] SeedAndFlags = new byte[seed.Length + FlagsBytes.Length];
+				Array.Copy(seed, SeedAndFlags, seed.Length);
+				Array.Copy(FlagsBytes, 0, SeedAndFlags, seed.Length, FlagsBytes.Length);
+				byte[] hash = hasher.ComputeHash(SeedAndFlags);

```

Comment:
I think you can do something like:

    flagsBytes = Blob.FromBase64(flagstring);
    hash = hasher.ComputeHash(flagsBytes + seed);